test/filemagic_test.rb

```diff
@@ -133,7 +139,6 @@ def test_compile
     res = fm.compile(path_to('perl'))
     fm.close
     assert(res)
-    File.unlink(path_to('perl.mgc', '.'))
```

Comment:
:+1: But it might be unnecessary code, so can I create separated PR or commit instead of just remove it?